pkg/plugin/test-keeper/metrics.go

```diff
@@ -0,0 +1,72 @@
+package testkeeper
+
+import (
+	"github.com/arquillian/ike-prow-plugins/pkg/log"
+	gogh "github.com/google/go-github/github"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/arquillian/ike-prow-plugins/pkg/server"
+)
+
+const (
+	// WithTests is a label used in prometheus metrics pull_requests_total for pull request containing some tests.
+	WithTests = "with_tests"
+	// WithoutTests is a label used in prometheus metrics pull_requests_total for pull request without any tests.
+	WithoutTests = "without_tests"
+)
+
+var (
+	pullRequestsCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "pull_requests_total",
+		Help: "Total number of pull requests.",
+	}, []string{"full_name", "type"})
+	okWithoutTestsPullRequest = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "pull_requests_changed_file_size",
+		Help:    "Histogram for changed file size of pull request with ok-without-tests.",
+		Buckets: prometheus.ExponentialBuckets(1, 5, 6),
+	}, []string{"full_name"})
+)
+
+// RegisterMetrics registers prometheus collectors to collect metrics for test-keeper.
+func RegisterMetrics() []error {
+	errors := make([]error, 0, 2)
+	server.RegisterOrGet(pullRequestsCounter, &errors, func(collector prometheus.Collector) {
+		pullRequestsCounter = collector.(*prometheus.CounterVec)
```

Comment:
I'm a bit confused now. In the general `metrics` https://github.com/arquillian/ike-prow-plugins/blob/master/pkg/server/metrics.go#L32 we assign it to the variable inside of the object. Here it is assigned to a global variable. 
I'd like to have it consistent or is there any difference in the usage?

Reply:
this is not a global variable, we are assigning it as an unexported variable. no there is nothing difference in usage.

Reply:
unexported global variable :-). global variable in the context of this file - never mind - the main point was that the code/usage is inconsistent